jina/drivers/encode.py

```diff
@@ -35,3 +35,13 @@ def __call__(self, *args, **kwargs):
                     'the first dimension must be the same' % (len(chunk_pts), embeds.shape))
             for c, emb in zip(chunk_pts, embeds):
                 c.embedding.CopyFrom(array2pb(emb))
+
+
+class UnaryEncodeDriver(EncodeDriver):
+    """The :class:`UnaryEncodeDriver` extracts the chunk-level content from documents and copies
+        the same content to the embedding
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._method_name = None
+        self._exec_fn = lambda x: x
```

Comment:
What does this do? Looks like a NOP or placeholder?

Reply:
it will mean a unary encoder as an executor function to be called by the driver

Reply:
The usage is similar as the `UnarySegmentDriver`. This is useful for testing and debugging.